libs/datamodel/core/src/transform/dml_to_ast/generator_serializer.rs

```diff
@@ -0,0 +1,58 @@
+use crate::{ast, configuration::Generator};
+
+pub struct GeneratorSerializer {}
+
+impl GeneratorSerializer {
+    pub fn add_generators_to_ast(generators: &[Generator], ast_datamodel: &mut ast::SchemaAst) {
+        let mut tops: Vec<ast::Top> = Vec::new();
+
+        for generator in generators {
+            tops.push(ast::Top::Generator(Self::lower_generator(&generator)))
+        }
+
+        // Prepend generators.
+        tops.append(&mut ast_datamodel.tops);
```

Comment:
nitpick: did you mean "Append generators"?

Reply:
It actually indeed meant prepend. Also had to read the code to really understand. Tried to make the comment clearer.

Reply:
Ok yes the new comment makes sense :)